### PR TITLE
IP 29

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: unabridged-works-volume-2/original/29-letters.adoc [hunk 1]
```diff
@@ -91,17 +97,18 @@ thine eyes may be opened to see the way of righteous government in the true ligh
...
+[.signed-section-context-close]
+Aylesbury prison, where I am visiting some of my dear Friends in God`'s eternal truth,
+17th 7th mo.
 1661
 
 [.postscript]
 ====
 
-Postscript.--Let thy government be like unto God`'s: even a yoke to the unjust,
+Postscript. Let thy government be like unto God`'s: even a yoke to the unjust,
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/29-letters.adoc:111]: This is sort of an archaic combination of punctuation, but I don't consider it an error (in fact, it's the punctuation that the linter recommends when a postcript starts with "Postscript". You don't need to change this one, but probably going forward you can leave these alone. 👍

File: unabridged-works-volume-2/original/29-letters.adoc [hunk 2]
```diff
@@ -978,15 +1055,20 @@ O fear the Lord, and watch over one another in love, faithfulness,
...
 
+[.signed-section-signature]
 I+++.+++ P.
 
+[.signed-section-context-close]
 27th of 11th Month, 1666
 
-TO THE SINGLE, UPRIGHT-HEARTED, AND FAITHFUL FRIENDS OF TRUTH, IN AND ABOUT THE TWO CHALFONTS
+[.letter-heading]
+To the Single, Upright-hearted, and Faithful Friends of Truth, in and about the Two CHalfonts
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/29-letters.adoc:1068]: Whoops, we got a funky `H` in `CHalfonts`

Comment (maijapapaya) [reply unabridged-works-volume-2/original/29-letters.adoc:1068]: I'm really trying to notice these before I send the PR!! Ugh!!

**Comment by jaredh159** [comment]
Thanks, merging! :)

**Pull request merged**